js/tests/unit/tab.js

```diff
@@ -1,5 +1,6 @@
+/* global QUnit */
 $(function () {
-  'use strict';
+  'use strict'
```

Comment:
It's been awhile since I did this. most likely just following eslint prompts in atom